mora/service/orgunit.py

```diff
@@ -473,14 +473,25 @@ def create_org_unit():
 
     name = common.checked_get(req, keys.NAME, "", required=True)
 
-    parent = common.checked_get(req, keys.PARENT, {}, required=True)
-    parent_uuid = common.get_uuid(parent)
+    parent_uuid = common.get_mapping_uuid(req, keys.PARENT, required=True)
     organisationenhed_get = c.organisationenhed.get(parent_uuid)
-    org_uuid = organisationenhed_get['relationer']['tilhoerer'][0]['uuid']
 
-    org_unit_type = common.checked_get(req, keys.ORG_UNIT_TYPE, {},
-                                       required=True)
-    org_unit_type_uuid = common.get_uuid(org_unit_type)
+    if organisationenhed_get:
+        org_uuid = organisationenhed_get['relationer']['tilhoerer'][0]['uuid']
+    else:
+        organisation_get = c.organisation(uuid=parent_uuid)
+
+        if organisation_get:
+            org_uuid = parent_uuid
+        else:
+            raise exceptions.HTTPException(
+                exceptions.ErrorCodes.V_ORG_NOT_FOUND,
```

Comment:
I don't think this error message is descriptive enough here, as we'll also reach this point if an org unit isn't found. Maybe a new error message showing that we did a lookup on `parent_uuid` in both `organisation` and `organisationenhed`?

Reply:
Also, the docstring should be updated to reflect this.